### PR TITLE
Add note about customizing modal-popup

### PR DESCRIPTION
**Pull request by samselikoff**

### CODE REVIEW

File: tests/dummy/app/templates/modal-documentation/modal.hbs [hunk 1]
```diff
@@ -79,5 +79,7 @@ withParams: ["foo", {
 </dl>
 
 <div class="callout callout-info">We still need an API for customizing
-which animated transitions to use for showing/hiding the dialog
-box.</div>
+which animated transitions to use for showing/hiding your various dialog
+boxes. For now, you can override the default by defining a 
+{{#link-to "transitions-documentation.defining"}}custom animation{{/link-to}}
```
Comment (ef4) [inline tests/dummy/app/templates/modal-documentation/modal.hbs]: I think this should be `transitions.defining`

**Comment by ef4** [comment]
Thanks, this is very helpful. Please double check the link that I commented above and then I can merge.

**Comment by samselikoff** [comment]
will do. Should we lock ember to 1.81 in the app? `bower install` is giving me a conflict over handlebars 1.3 vs 2.0

**Comment by ef4** [comment]
Try running `EMBER_CHANNEL=1.8.1 ./packaging/select-ember-version.js`

That's how Travis switches between Ember channels. It also accepts "release", "beta", and "canary".

**Comment by ef4** [comment]
But yes, possibly we should lock the default setup so it runs out of the box.

**Comment by samselikoff** [comment]
verified locally, should be all set

**Comment by samselikoff** [comment]
ok, now all set. (custom animation -> custom transition)

**Comment by ef4** [comment]
Thanks.

**Pull request merged**